contracts/upgradeable_contracts/erc20_to_native/FeeManagerErcToNative.sol

```diff
@@ -25,4 +25,8 @@ contract FeeManagerErcToNative is BaseFeeManager {
             (new Sacrifice).value(_fee)(_rewardAddress);
         }
     }
+
+    function getAmountToBurn(uint256 _value, uint256 _fee) public pure returns(uint256) {
```

Comment:
I don't think that it make sense to specify `_fee` here since the fee manager already now how to calculate the fee for this scenario.

Reply:
Fixed here https://github.com/poanetwork/poa-bridge-contracts/pull/164/commits/f7cb8d253e4418082a6f1230b36c6f0f4270e463. I had to update `calculateFee` from `external` to `public` to be able to call it from `getAmountToBurn`

Reply:
Thanks